zebra/zebra_vty.c

```diff
@@ -833,6 +857,7 @@ DEFPY(ip_route_address_interface_vrf,
 	  tag (1-4294967295)                           \
 	  |(1-255)$distance                            \
 	  |label WORD                                  \
+	  |table (1-4294967295)                        \
```

Comment:
Again, specifying a table inside of a vrf ip route?  Makes no sense

Reply:
this parameter will be handled only in the case with vrf backend = VRF netns